packages/storybook/stories/va-loading-indicator.stories.jsx

```diff
@@ -3,13 +3,14 @@ import React, { useState } from 'react';
 import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
 
 const loadingIndicatorDocs = getWebComponentDocs('va-loading-indicator');
+const componentName = 'Loading indicator';
 
 export default {
-  title: 'Components/va-loading-indicator',
+  title: `Components/${componentName}`,
   parameters: {
-    componentSubtitle: `Loading indicator web component`,
+    componentSubtitle: `va-loading-indicator web component`,
     docs: {
-      page: () => <StoryDocs data={loadingIndicatorDocs} />,
+      page: () => <StoryDocs componentName={componentName} data={loadingIndicatorDocs} />,
```

Comment:
If we modify each story default export to include an `id` property, we won't have to update links on design.va.gov.

```jsx
export default {
  id: 'components/va-loading-indicator', // results in http://design.va.gov/storybook/?path=/docs/components-va-loading-indicator
  title: `Components/${componentName}`,
  parameters: {
    componentSubtitle: `va-loading-indicator web component`,
    docs: {
      page: () => <StoryDocs componentName={componentName} data={loadingIndicatorDocs} />,
...
```